### DISCUSSION TITLE
Repocket error peer-client-version

### DISCUSSION THREAD

**Discussion by nkciam** [engageub/InternetIncome#187] [General]
Hi engageub,
Currently, I tried to run Repocket with proxy, but seems there was an issue with Repocket and it continues restarting. Could you help me check? Thanks! 

Repocket logs:
![image](https://github.com/user-attachments/assets/e242ee6b-0383-49f1-b8e2-1b634d7cab10)

![image](https://github.com/user-attachments/assets/e2467268-70d2-47b8-97ec-145b29908ff6)

tun2sock logs:
![image](https://github.com/user-attachments/assets/9e229e6e-3922-4170-b811-7ff2dff9ea4d)

*Using tun2proxy instead of tun2sock working fine
![image](https://github.com/user-attachments/assets/2a47ecf7-a085-4540-962c-13c0c4ac75a9)

**Comment by engageub** [comment]
Hello,
Thank you for sharing the logs.
It appears that you are using the latest version of Repocket, while the client is using an older version.
Please note that the official Docker Repocket image is being used.
If there are any issues, we will need to contact the Repocket team.
Upon reviewing the tun2socks logs, there are timeouts for one of the IP addresses, which is not reachable via ping commands. There may be an issue with their server.
Other users have posted Repocket errors in the Discord channel, mentioning that one of the containers was showing the error.
The script is working as expected; the issue may lie with Repocket.

![image](https://github.com/user-attachments/assets/1df7a759-0f28-487e-a1c2-0096b02ba74c)


Thank you

**Reply by nkciam** [reply]
Working fine with latest version branch main now. Thank you!

**Comment by TradeBW** [comment]
Has anyone lately observed significant drop in the repockets revenue/earnings ? I saw 50% drop with same IP quality and count. 
Geo is Tier 1 so I don't that as an issue.

**Reply by engageub** [reply]
Hi,
Could you please join repocket discord team.  It will give you better understanding if there is any issue with repocket application or their traffic. 

Thank you

**Reply by TradeBW** [reply]
Did it twice, no one is responding, hence messaged here

**Reply by engageub** [reply]
Hi,
Official Docker Image of the app is being used. If there are any network issues you have to look into tun2socks logs.
If there are any application issues,  you have to look into repocket docker logs.
Traffic distribution is done by repocket and it depends on demand and your IP quality and the number of devices online.

Thank you

**Comment by TradeBW** [comment]
Everything is working fine.

I am not asking about errors.

It is just general query if anyone else has faced earnings drop.

My IPs are Tier 1, residential. This has happened in last 3-4 days since
they launched new interface.

**Reply by engageub** [reply]
I understand that but lets wait if someone else answers.

**Comment by techroy23** [comment]
i got that too. posibly rate limited.
if you have portainer installed. stop those containers for atleast 1hr or more.
